cmd/format/format.go

```diff
@@ -154,12 +150,32 @@ func Run(v *viper.Viper, statz *stats.Stats, cmd *cobra.Command, paths []string)
 		formatters[name] = formatter
 	}
 
-	// open the cache if configured
+	var db *bolt.DB
+
 	if !cfg.NoCache {
-		if err = cache.Open(cfg.TreeRoot, cfg.ClearCache, formatters); err != nil {
-			// if we can't open the cache, we log a warning and fallback to no cache
-			log.Warnf("failed to open cache: %v", err)
-			cfg.NoCache = true
+		// open the db
+		db, err = cache.Open(cfg.TreeRoot)
+		if err != nil {
+			return fmt.Errorf("failed to open cache: %w", err)
+		}
+
+		// ensure db is closed after we're finished, and removed if temporary
```

Comment:
What does "temporary" refer to here? Has this comment bitrotted?

Reply:
Updated it :)